docs/development.rst

```diff
@@ -554,3 +554,23 @@ following to your ``.env`` file::
     MDN_API_S3_BUCKET_NAME=<your-s3-bucket-name>
     AWS_ACCESS_KEY_ID=<your-aws-access-key>
     AWS_SECRET_ACCESS_KEY=<your-aws-secret-key>
+
+
+Enabling ``django-querycount``
+==============================
+
+If you want to find out how many SQL queries are made, per request,
+even if they are XHR requests, you can simply add this to your ``.env`` file:
```

Comment:
@peterbe Add an extra `:` to the end of this line (so ` file::`) to get the code block below into the proper format.